src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -641,9 +692,64 @@ export class ServiceFormComponent extends CdForm implements OnInit {
       // osd       - This is deployed a different way.
       // container - This should only be used in the CLI.
       // promtail  - This is deprecated and replaced by alloy.
-      this.hiddenServices.push('osd', 'container', 'promtail');
+      this.hiddenServices.push('osd', 'container', 'promtail', 'mon', 'mgr');
+
+      const rawTypes = _.difference(resp, this.hiddenServices).sort();
+      this.serviceTypes = rawTypes;
+      this.serviceTypeOptions = rawTypes.map((type) => ({
+        value: type,
+        label: this.SERVICE_TYPE_LABELS[type] || _.startCase(type)
+      }));
+
+      // Build ordered groups
+      const serviceGroups: Array<{ label: string; types: string[] }> = [
+        { label: 'Block', types: ['rbd-mirror', 'iscsi', 'nvmeof'] },
+        { label: 'Object', types: ['rgw'] },
+        { label: 'File', types: ['mds', 'cephfs-mirror', 'nfs', 'smb'] },
+        { label: 'Logs & Observability', types: ['loki', 'alloy'] },
+        {
+          label: 'Monitoring',
+          types: [
+            'grafana',
+            'prometheus',
+            'alertmanager',
+            'snmp-gateway',
+            'ceph-exporter',
+            'node-exporter',
+          ]
+        },
+        { label: 'Security & SSO', types: ['mgmt-gateway', 'oauth2-proxy'] },
 
-      this.serviceTypes = _.difference(resp, this.hiddenServices).sort();
+      ];
```

Comment:
are we removing mgr and mon entirely from the list? because we still can edit them and it looks like broken. This is still useful to scale up or down.

<img width="879" height="329" alt="Image" src="https://github.com/user-attachments/assets/b19500c1-dba1-4152-b3fd-546295e37b37" />